README.md

```diff
@@ -1 +1,159 @@
+
+**Solver** is a tool that helps the with some cool functions
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+  <details>
+    <ul>
+      <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    </ul>
+  </details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **Test functions**
+
+## Getting Started
+
+### Prerequisites
+In order to run this project, you must have installed and configured ruby in your local environment:
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git@github.com:MasumaJaffery/Enumerable.git
+```
+ 
+--->
+Before to run the program, verify that you have Ruby installed on your OS running the following command:
+<!--
+```sh
+   ruby -v
+```
+--->
+
+### Usage
+
+Run the following command inside the project folder:
+1. Open your terminal and navigate to the root directory of your project.
+2. Open the `lib` directory ```$ cd lib```
+3. Run ```$ ruby solver.rb```
+4. Enjoy
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+You can run this tests
+
+Run the following command inside the project folder:
+1. Open your terminal and navigate to the root directory of your project.
+2. Open the `spec` directory ```$ cd spec```
+3. Run ```$ rspec spec```
+4. To run tests
+
+
+### Deployment
+
+You can deploy this project using: GitHub Pages Netifly or Render 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Joseph L Kausi**
+
+- GitHub: [@Grandi0z](https://github.com/Grandi0z)
+- Twitter: [@Joseph_LSK](https://twitter.com/Joseph_LSK)
+- LinkedIn: [Joseph L Kausi](https://www.linkedin.com/in/joskal/)
+
+👤 **Glauber Bannwart**
+
+- GitHub: [@cancelei](https://github.com/cancelei)
+- LinkedIn: [Glauber Bannwart](https://www.linkedin.com/in/gbannwart/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Add more tests**
+- [ ] **Add more functions**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Grandi0z/solver/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, I would like to Thank You!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Special Thanks to the Microverse and Micronauts.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]() licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] - [ ] Please kindly fix this link, I was redirected to a wrong page; implementing this will make your README file to be more engaging. ℹ️ You will need to create a license file, this [template](https://choosealicense.com/licenses/mit/) might be usefull. 🙏